Update head components section at creating_themes.rst (#248)
mautic/user-documentation: docs/builders/creating_themes.rst

.. vale on

> Mautic processes most of the ``<mj-head>`` components. ``<mj-attributes>`` do not run.

Reviewer: ```suggestion
Mautic processes most of the ``<mj-head>`` components. ``<mj-attributes>`` don't run.
```

Small fix from our Vale linter to bring grammar in line with the style guide.

> **Tested elements** are: mj-breakpoint, mj-font, mj-html-attributes, mj-style, mj-title, mj-preview 

Reviewer: ```suggestion
**Tested elements** include: ``mj-breakpoint``, ``mj-font``, ``mj-html-attributes``, ``mj-style``, ``mj-title``, and ``mj-preview``.
```

Let's make sure these are all enclosed in inline code blocks. This will prevent the Vale warning about HTML and also makes it clear we're referring to code-related terms.

Author: Thanks, I will add these changes

Author: Committed changes
